### PR TITLE
Updated KR translations

### PR DESCRIPTION
**Pull request by psp0**
hello~ 
I made some changes.

- I unified meal coverage-related terms into ‘식재료 충족치’.
- I know the event is in progress but no matter how much I think about it, the 'rate' does not correspond to the 'multiplier,' so I change it to '배수' (multiple).
- I know the Override means, but I don't know how to describe in korean. so I have to translate 신규('new'). 
->I'm just curious!😉 In English-speaking countries, is the word override used often when it has nothing to do with programming? 
- I change the word 자는 중 (2) ('asleep (2)') to 두 번째 잠(second sleep), because In the  Korean community(pksleep), many people usually say 2잠(second sleep).

thanks!!!

### CODE REVIEW

**Comment by RaenonX** [comment]
> hello~ I made some changes.
> 
> * I unified meal coverage-related terms into ‘식재료 충족치’.
> * I know the event is in progress but no matter how much I think about it, the 'rate' does not correspond to the 'multiplier,' so I change it to '배수' (multiple).
> * I know the Override means, but I don't know how to describe in korean. so I have to translate 신규('new').
>   ->I'm just curious!😉 In English-speaking countries, is the word override used often when it has nothing to do with programming?
> * I change the word 자는 중 (2) ('asleep (2)') to 두 번째 잠(second sleep), because In the  Korean community(pksleep), many people usually say 2잠(second sleep).
> 
> thanks!!!

> I know the Override means, but I don't know how to describe in korean. so I have to translate 신규('new').
->I'm just curious!😉 In English-speaking countries, is the word override used often when it has nothing to do with programming?
To my understanding, sometimes **overwrite**, but that sounds a bit weird here. I don't really have any ideas of what could be a better word tbh.

> I change the word 자는 중 (2) ('asleep (2)') to 두 번째 잠(second sleep), because In the Korean community(pksleep), many people usually say 2잠(second sleep).
If Korean Community usually say 2잠, then maybe good to put it as just 2잠?

------------------

Imma wait for your response before rebasing then merging then

**Comment by RaenonX** [comment]
I think you want to rebase onto the current main instead

Pushed a test commit to be wiped out later tho, so maybe don't rebase now, unless you are rebasing onto 9313b258. 

image

**Comment by psp0** [comment]
i'm trying:)

**Comment by RaenonX** [comment]
> i'm trying:)

This is good, except that remember to drop the commit 7018067d3a2072c542e9609c8bde2e7b1eb3f411 or rebase again

**Comment by RaenonX** [comment]
Also let me know if `game-*.json` have wrong translation. It's automatically generated so even if you edit that, it will get overwrote someday.

**Comment by psp0** [comment]
In this rebase, I am using only Git commands, not a GUI, so I encountered some struggles. Thanks for your patience. I edited game-kr.json because I noticed other languages on the KR website, which caught my attention. 🤗

**Comment by RaenonX** [comment]
> In this rebase, I am using only Git commands, not a GUI, so I encountered some struggles. Thanks for your patience. I edited game-kr.json because I noticed other languages on the KR website, which caught my attention. 🤗

Using CLI is better, GUI got limited features anyway.

File: messages/game-kr.json [hunk 1]
```diff
@@ -1023,4 +1023,4 @@
     "onBedTime": "잠자기 약속을 {num}회 지키자",
     "catchPokemon": "포켓몬 {num}마리와 동료가 되자"
   }
-}
\ No newline at end of file
+}
```
Comment (psp0) [inline messages/game-kr.json:1026]: I don't know how to change these things.. may be vscode setting i guess..

Comment (RaenonX) [reply messages/game-kr.json:1026]: Yeah that's likely VSCode settings, I would just amend the commit to not to include this change tho

**Pull request merged**